code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -37,6 +37,9 @@
 	cuff(C, user)
 
 /obj/item/restraints/handcuffs/proc/cuff(mob/living/carbon/C, mob/user, remove_src = TRUE)
+	if(!istype(C)) // Shouldn't be able to cuff anything but carbons.
```

Comment:
You can remove line 25-26 now this check is in place. The regular cuffs also did a check.

Reply:
oop yeah, missed that

Reply:
Also turns out I need that one on line 25-26 because there's some stuff in that proc that needs the user to be a carbon. But I also need this one because borgs call `cuff` directly.. its a bit of a mess, could probably use a refactor but I really don't want to do that right now.